content/en/docs/instrumentation/swift/automatic.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Automatic Instrumentation
```

Comment:
Let's call this "Instrumentation Libraries" for now, otherwise this is misleading, see https://github.com/open-telemetry/opentelemetry-specification/issues/2866

```suggestion
title: Instrumentation Libraries
```

Reply:
In that case the file should be renamed too. What should the new name be @cartermp @svrnm (so that were somewhat consistent with other language SIGs)?